examples/tera_template/README.md

```diff
@@ -0,0 +1,63 @@
+# Tera Template
+
+Am example usage of Tera template engine working with Gotham.
```

Comment:
Small typo of "Am". It would also be nice to get a link to the Tera project here too.